apps/consent-ui/src/components/Form/fieldsets/FieldSet/index.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { FieldValues } from 'react-hook-form';
+import clsx from 'clsx';
+
+import RequiredAsterisk from '../../RequiredAsterisk';
+import { FormFieldSetProps } from '../../types';
+
+const FieldSet = <T extends FieldValues>({
+	children,
+	className,
+	error,
+	label,
+	name,
+	required = false,
+}: FormFieldSetProps<T>) => {
+	return (
+		<fieldset className={clsx('fieldset', className)}>
+			<label htmlFor={name}>
+				{label}
+				{required && <RequiredAsterisk />}
+			</label>
+			{children}
+			{error && <p style={{ color: 'red' }}>{error}</p>}
```

Comment:
Not asking for change, just sharing my own practices that I've found useful:
In layouts that include multiple different children/components, I tend to add blank lines to facilitate visually parsing the layout from source.

```suggestion
			</label>

			{children}

			{error && <p style={{ color: 'red' }}>{error}</p>}
```